source/jormungandr/jormungandr/street_network/handimap.py

```diff
@@ -85,7 +82,7 @@ def __init__(
         self.headers = {"Content-Type": "application/json", "Accept": "application/json"}
         self.timeout = timeout
         self.modes = modes if modes else ["walking"]
-        self.language = self._get_language(kwargs.get('language', "french"))
+        self.language = self._get_language(kwargs.get('language', "fr-FR"))
```

Comment:
Shouldn't you default to `None` here, and then in `_get_language` use the one from the instance in case of `None`?

Reply:
:+1: